crates/aranya-crypto-ffi/Cargo.toml

```diff
@@ -41,8 +41,8 @@ testing = [
 ]
 
 [dependencies]
-aranya-crypto = { version = "0.4.0", path = "../aranya-crypto", default-features = false, features = ["alloc"] }
-aranya-policy-vm = { version = "0.5.0", path = "../aranya-policy-vm", default-features = false, features = ["derive"] }
+aranya-crypto = { version = "0.5.0", path = "../aranya-crypto", default-features = false, features = ["alloc"] }
```

Comment:
Why do we use `0.x.0` everywhere? We should just be using `0.x` so that we can follow patches (bug fixes).